MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -1780,7 +1781,7 @@ public void AddStudio(string name)
 
         public void SetStudios(IEnumerable<string> names)
         {
-            Studios = names.Distinct().ToArray();
+            Studios = names.Trimmed().Distinct().ToArray();
```

Comment:
`Trimmed()` against an `IEnumerable<string>` returns an enumeration of the values where each is `Trim()`ed.